web/packages/teleterm/src/mainProcess/runtimeSettings.ts

```diff
@@ -39,7 +39,7 @@ const TSH_BIN_ENV_VAR = 'CONNECT_TSH_BIN_PATH';
 const TSH_BIN_DEFAULT_PATH_FOR_DEV = path.resolve(
   __dirname,
   '..', '..', '..', '..', '..', '..',
-  'build', 'tsh',
+  'build', process.platform === 'win32' ? 'tsh.exe' : 'tsh',
```

Comment:
Until this is merged, dev version of the app must be started with `CONNECT_TSH_BIN_PATH=$PWD/build/tsh.exe.`